aten/src/ATen/native/native_functions.yaml

```diff
@@ -6642,6 +6642,16 @@
   structured_delegate: _addmm_activation.out
   variants: function, method
 
+- func: _scaled_mm(Tensor self, Tensor mat2, *, ScalarType? dtype=None, Tensor? scale_a=None, Tensor? scale_b=None, Tensor? scale_result=None) -> (Tensor, Tensor)
```

Comment:
What is the dtype argument doing here? If it just for output dtype: `out_dtype` would be a better name>.